doc/man1/openssl-sess_id.pod

```diff
@@ -32,18 +35,13 @@ properly, most users will not need to use it.
 
 Print out a usage message.
 
-=item B<-inform> B<DER>|B<PEM>
-
-This specifies the input format. The B<DER> option uses an ASN1 DER encoded
-format containing session details. The precise format can vary from one version
-to the next.  The B<PEM> form is the default format: it consists of the B<DER>
-format base64 encoded with additional header and footer lines.
+=item B<-inform> B<DER>|B<PEM>, B<-outform> B<DER>|B<PEM>|B<NSS>
 
-=item B<-outform> B<DER>|B<PEM>|B<NSS>
+The input and output formats; the default is PEM..
+See L<openssl(1)/Format Options> for details.
 
-This specifies the output format. The B<PEM> and B<DER> options have the same
-meaning and default as the B<-inform> option. The B<NSS> option outputs the
-session id and the master key in NSS keylog format.
+For B<NSS> output, the session ID and master key are reported in NSS "keylog"
```

Comment:
the NSS .... format?

Reply:
I don't understand what you mean, sorry.

Reply:
I meant isn't there missing 'the'? But of course I might be wrong as non-native speaker coming from language that does not have definite and indefinite articles at all.

Reply:
No, it's correct. I  can't think of an example other than something like "report in PEM format"  And yes, it's indefinite.

Your English is better than my non-existant Czech.